docs/custom_conf.py

```diff
@@ -0,0 +1,201 @@
+import datetime
+
+# Custom configuration for the Sphinx documentation builder.
+# All configuration specific to your project should be done in this file.
+#
+# The file is included in the common conf.py configuration file.
+# You can modify any of the settings below or add any configuration that
+# is not covered by the common conf.py file.
+#
+# For the full list of built-in configuration values, see the documentation:
+# https://www.sphinx-doc.org/en/master/usage/configuration.html
+
+############################################################
+### Project information
+############################################################
+
+# Product name
+project = 'Ubuntu packaging'
+author = 'Canonical Group Ltd.'
+version = '2.0'
+
+# Uncomment if your product uses release numbers
+# release = '1.0'
+
+# The default value uses the current year as the copyright year.
+#
+# For static works, it is common to provide the year of first publication.
+# Another option is to give the first year and the current year
+# for documentation that is often changed, e.g. 2022–2023 (note the en-dash).
+#
+# A way to check a GitHub repo's creation date is to obtain a classic GitHub
+# token with 'repo' permissions here: https://github.com/settings/tokens
+# Next, use 'curl' and 'jq' to extract the date from the GitHub API's output:
+#
+# curl -H 'Authorization: token <TOKEN>' \
+#   -H 'Accept: application/vnd.github.v3.raw' \
+#   https://api.github.com/repos/canonical/<REPO> | jq '.created_at'
+
+copyright = '%s, %s' % (datetime.date.today().year, author)
+
+## Open Graph configuration - defines what is displayed in the website preview
+# The URL of the documentation output
+ogp_site_url = 'https://canonical-ubuntu-packaging-guide.readthedocs-hosted.com/'
+# The documentation website name (usually the same as the product name)
+ogp_site_name = 'Ubuntu Packaging Guide'
+# An image or logo that is used in the preview
+ogp_image = 'https://assets.ubuntu.com/v1/253da317-image-document-ubuntudocs.svg'
+
+# Update with the favicon for your product (default is the circle of friends)
+html_favicon = '.sphinx/_static/favicon.png'
+
+html_title = 'Ubuntu Packaging Guide'
+
+# (Some settings must be part of the html_context dictionary, while others
+#  are on root level. Don't move the settings.)
+html_context = {
+
+    # Change to the link to your product website (without "https://")
+    'product_page': 'packages.ubuntu.com/',
+
+    # Add your product tag to ".sphinx/_static" and change the path
+    # here (start with "_static"), default is the circle of friends
+    'product_tag': '_static/tag.png',
+
+    # Change to the discourse instance you want to be able to link to
+    # using the :discourse: metadata at the top of a file
+    # (use an empty value if you don't want to link)
+    'discourse': 'https://discourse.ubuntu.com/c/foundations/',
```

Comment:
issue: This should not be be linking to foundations. The product is Ubuntu Packaging/Development this is nothing foundations specific.

suggestion: Linking to https://discourse.ubuntu.com/ should be enough:
```suggestion
    'discourse': 'https://discourse.ubuntu.com/',
```

Reply:
Ack.

Reply:
I'd probably flag this as another place where we might be able to link back to the Community team, who has their own Discourse. We should check with them what Discourse they think would be the best to drive traffic to before this goes live. Doesn't have to be resolved in this PR, but pointing just to the generic discourse.ubuntu might leave people feeling a bit confused about where they should be posting (the landing page for that discourse can be a bit overwhelming).